sympy/integrals/laplace.py

```diff
@@ -997,6 +1000,45 @@ def _laplace_apply_simple_rules(f, t, s):
     return None
 
 
+def _piecewise_to_heaviside(f, t):
+    """
+    This function converts a Piecewise expression to an expression written
+    with Heaviside. It is not exact, but valid in the context of the Laplace
+    transform.
+    """
+    tr = symbols('tr', real=True)
```

Comment:
```suggestion
    tr = Dummy('tr', real=True)
```
It's suggested to use Dummy symbols as much as possible, because user may pick `tr` as symbol name

Reply:
That I should have remembered; sorry for the additional review work.